### PR TITLE
Add Dockerfile for backend

### PR DESCRIPTION
**Pull request by pbzweihander**
@tirr-c Can we merge frontend Dockerfile to this?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+FROM lukemathwalker/cargo-chef:latest-rust-1.70.0-bullseye AS chef
```
Comment (tirr-c) [inline Dockerfile]: Current Rust version is 1.71.1, is this ok?

Comment (pbzweihander) [reply Dockerfile]: oh, i'll update

**Comment by tirr-c** [comment]
> @tirr-c Can we merge frontend Dockerfile to this?

Maybe we can do that, runtime base image would be `node` in that case.

**Comment by pbzweihander** [comment]
that's fine

**Review by tirr-c — APPROVED** [review]

**Pull request merged**